rcl/test/rcl/test_guard_condition.cpp

```diff
@@ -0,0 +1,17 @@
+// Copyright 2015 Open Source Robotics Foundation, Inc.
```

Comment:
This file was left over from a pull request from last year. Which brings up the question of when the cr should be set, the year you start the file or when it is committed.

I can also remove this file since there are no tests here yet.